src/scalar_store.rs

```diff
@@ -216,6 +222,33 @@ pub enum ScalarExpression<F: LurkField> {
     UInt(UInt),
 }
 
+#[cfg(not(target_arch = "wasm32"))]
+impl<Fr: LurkField> Arbitrary for ScalarExpression<Fr> {
+    type Parameters = ();
+    type Strategy = BoxedStrategy<Self>;
+
+    fn arbitrary_with(_args: Self::Parameters) -> Self::Strategy {
```

Comment:
Minor quibble: in the future we probably don't want to have this Strategy be the default Arbitrary `impl` in favor of something that generates well-formed `ScalarExpression`s. This one here is great for testing serialization/hashing, but not so good for testing parsing/printing/evaluation since the inner `ScalarPtr`s will be noise